Changelog.md

```diff
@@ -1,3 +1,10 @@
+### Development
+
+Bug Fixes:
+
+* Correctly run `before(:suite)` (and friends) in the context of an example
```

Comment:
There are multiple (`before`, `after`, `prepend_before`, `append_before`, `prepend_after`, `append_after`) but I'll change it if you prefer.